logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -45,12 +45,42 @@ import com.wire.kalium.logic.data.user.Connection
 import com.wire.kalium.logic.data.user.SupportedProtocol
 import com.wire.kalium.logic.data.user.UserId
 import com.wire.kalium.logic.logStructuredJson
+import com.wire.kalium.logic.sync.incremental.EventSource
 import com.wire.kalium.network.api.base.authenticated.conversation.ConversationResponse
 import com.wire.kalium.util.DateTimeUtil
 import com.wire.kalium.util.serialization.toJsonElement
 import kotlinx.serialization.json.JsonNull
 
-sealed class Event(open val id: String, open val transient: Boolean, open val live: Boolean) {
+/**
+ * A wrapper that joins [Event] with its [EventDeliveryInfo].
+ */
+data class EventEnvelope(
```

Comment:
does this need to have a `toLogMap()` to properly show up in the logs?

Reply:
Yup! Nice catch.
When I refactored I ended up logging only the `event`, so it would keep working. But by doing it, `isTransient` and `isLive` were no longer being logged. Added `toLogMap()` and the like to both.